primeqa/text_classification/processors/postprocessors/text_classifier.py

```diff
@@ -0,0 +1,158 @@
+from typing import List, Dict, Any
+
+from datasets import Dataset
+import numpy as np
+import logging
+from transformers import EvalPrediction
+
+from primeqa.mrc.processors.postprocessors.abstract import AbstractPostProcessor
+from primeqa.mrc.data_models.eval_prediction_with_processing import EvalPredictionWithProcessing
+
+
+logger = logging.getLogger(__name__)
+
+
+class TextClassifierPostProcessor(AbstractPostProcessor):
+    """
+    Post processor for use with the text classifier
+    """    
+    def __init__(self, 
+                drop_label: str,
+                id_key: str,
+                label_list: List[str],
+                output_label_prefix: str,
+                *args, **kwargs):
+        """
+        Args:
+            drop_label: if specified, ignore this category for classifier output,
+                     e.g. "no_answer" converts a ("yes","no_answer","no") classifier into a ("yes", "no") classifier
+            id_key: unique identifier field of examples to be classified
+            label_list: the (human-readable) labels produced by the classifier
+            output_label_prefix: prefix for new output fields in eval_predictions.json
+            *args: Arguments for super class constructor.
+            **kwargs: Keyword Arguments for super class constructor.
+        """                
+        super().__init__(1, 1)
+        self.id_key=id_key
+        self.drop_label = drop_label
+        self.label_list = label_list
+        self.output_label_prefix = output_label_prefix
+
+
+    def process(self, examples: Dataset, features: Dataset, predictions: tuple):
```

Comment:
Do we need this abstract function here? there is another process() in this class which does all the work.